dashboard/api_test.py

```diff
@@ -232,6 +232,34 @@ def test_format_run_with_currently_enrolled(self):
             self.expected_ret_data
         )
 
+    def test_format_run_currently_enrolled_dont_display_progress(self):
```

Comment:
Can you also add a test for `CAN_UPGRADE`, or make this one parameterized?

Reply:
sure